changelog/0037-update-supervisor-confs.yml

```diff
@@ -0,0 +1,20 @@
+title: Run command to update Supervisor configurations
+key: update-supervisor-confs
+date: 2020-10-14
+optional_per_env: yes
```

Comment:
is this optional?

Reply:
No, thanks. https://github.com/dimagi/commcare-cloud/pull/4268/commits/1e0629c996e7cae15222816f583c7d2f6cfd1f40